Assignments/Program.cs

```diff
@@ -1 +1,53 @@
-﻿Console.WriteLine("Hello, World!");
+﻿// ---------------------------------------------------------------------------------------
+// T7 - Chess Board Printing
+// ---------------------------------------------------------------------------------------
+using System;
+using System.Collections.Generic;

```

Comment:
Not used.

Reply:
As Implicit Usings was enabled, these libraries were not used.  So, the implicit usings is disabled in the next commit.